infrastructure/restapi/src/main/java/tech/pegasys/teku/infrastructure/restapi/RestApiBuilder.java

```diff
@@ -197,7 +206,17 @@ private void configureCors(final JavalinConfig config) {
   }
 
   private Server createJettyServer() {
-    final Server server = new Server(InetSocketAddress.createUnresolved(listenAddress, port));
+    final Server server = new Server();
+    final ServerConnector connector;
+    if (maybeKeystorePath.isPresent()) {
+      connector = new ServerConnector(server, getSslContextFactory());
+    } else {
+      connector = new ServerConnector(server);
+    }
```

Comment:
This builder is currently only used by the validator api, so we won't be creating a non-ssl connector - if this api ends up in use by the beacon-api, then creating a non ssl version will be more important, but currently it's arguable that we don't even need the non ssl connector created here. Would be interested in your thoughts on removing this @ajsutton

Reply:
hmm, could remove it and it wouldn't be the end of the world as it's easy to add back.  Would only really be worth doing if `maybeKeystorePath` stopped being an `Optional` though.  That makes it slightly harder to support non-ssl again but still wouldn't be a big deal.

Given our aim is to move everything to this I'd be inclined to keep it since we have it unless it causes us any kind of trouble.

Reply:
I'll keep it - sounds fair :)